h/static/scripts/directive/excerpt.coffee

```diff
@@ -0,0 +1,12 @@
+module.exports = ->
+  link: (scope, elem, attr, ctrl) ->
+    scope.$evalAsync -> if elem[0].scrollHeight > elem[0].clientHeight
```

Comment:
This is strange indentation. I would prefer if you indent the rest of the block another level and put the `if` statement on its own line.